src/asciidoc/docs/publish/richPreview.adoc

```diff
@@ -18,15 +18,13 @@ You can also customize the sharing metadata by configuring an entitlement.
 To customize the sharing image for your template, render a custom view when the URL contains the `koji-screenshot=1` query parameter.
 For optimal rich preview support, design this view with the following considerations in mind.
 
-* Responsiveness.
-* Looks good with a square aspect ratio.
-* Includes customizable elements to generate unique previews of each remix.
+* The image is responsive.
+* The image looks good with a square aspect ratio.
+* The image is optimized. For example, you can use a third-party tool to reduce the image size to 400x400 pixels or load only the first frame of a gif.
```

Comment:
Might want to say what you mean by "optimized" (i.e., specifically what it's optimized for). Something about loading quickly.

Reply:
Fixed.